web-components/src/components/dropdown/scss/dropdown.scss

```diff
@@ -0,0 +1,138 @@
+:host(:focus) {
+  box-shadow: none !important;
+  outline: none !important;
+}
+
+:host {
+  .md-dropdown {
+    display: flex;
+    flex-direction: column;
+    position: relative;
+
+    &-select {
+      display: none;
+    }
+
+    &-label {
+      @include flex;
+      background-color: var(--md-primary-bg-color);
+      border: $dropdown-border;
+      border-radius: $dropdown-label-radius;
```

Comment:
The rounded pill shape is optional in all input fields - please refer to combobox and phone input and make it optional also.

Reply:
rolled-back